doc/specs/stdlib_constants.md

```diff
@@ -0,0 +1,21 @@
+---
+title: constants
+---
+
+[TOC]
+
+## Introduction
+
+
+The [[stdlib_constants]] module provides mathematical constants and the most common physical constants.
+
+The [[stdlib_codata]] module defines all codata constants as derived type [[codata_constant]].
+The lastest codata constants were released in 2018 by the [NIST](http://physics.nist.gov/constants.)
+The names are quite long and can be aliased with shorter names.
+
```

Comment:
The DT should be described in details here. There should be also a note that all provided values are `dp`.